lib/resty/cookie.lua

```diff
@@ -91,7 +94,10 @@ local function get_cookie_table(text_cookie)
     end
 
     if key ~= nil and value == nil then
-        cookie_table[key] = sub(text_cookie, i)
+        if not cookie_table[key] then
+            cookie_table[key] = {}
+        end
+        table.insert(cookie_table[key], sub(text_cookie, i))
```

Comment:
in the description we say that there is a list of values for each key, but here I see a table - am I missing sth?

Reply:
@muratozkan lists are tables in lua

Reply:
oh, lua does it again... thanks @alicerum